src/server.js

```diff
@@ -63,8 +63,8 @@ app.all('*', (req, res) =>
 );
 
 // finally, let's start our server...
-app.listen(process.env.PORT || 3000, () => {
-  console.log(`Listening on port`);
+const server = app.listen(process.env.PORT || 3000, () => {
```

Comment:
Why this?

Reply:
It was on develop before now, I edited it on my last PR that was merged so, I am just returning it back now

Reply:
I have removed it now